### DISCUSSION TITLE
Question about behavior of --build-from-source after "brew unlink gcc"

### DISCUSSION THREAD

**Discussion by suewonjp** [Homebrew/discussions#4912] [Everyday usage] [answered]
### Output of `brew config`

```shell
NA
```

### Output of `brew doctor`

```shell
NA
```

### Description of issue

This is just a simple question.

I have two gcc versions. One is from Homebrew and the other is a system gcc from native package manager (APT in Ubuntu).

If I do the following:
1. brew unlink gcc
2. brew install --build-from-source unzip

Then, will unzip be built with the system gcc?

**Comment by SMillerDev** [comment, marked as answer]
No, Homebrew builds with Homebrew packages

**Reply by suewonjp** [reply]
OK. Then Homebrew will forcefully execute `brew link gcc` without respecting `brew unlink gcc` by the user?

**Reply by SMillerDev** [reply]
No, it'll just use the unlinked gcc
